deploy/CMakeLists.txt

```diff
@@ -27,9 +26,21 @@ elseif(UNIX)
         )
     elseif(IS_DEBIAN)
         message(STATUS "Detected Debian version ${DEB_VERSION}")
-        list(APPEND CPACK_GENERATOR
-            DEB
-        )
+        # -- If cmake flag -DCREATE_RPM_FOR_UBUNTU is set to some
+        # value, then we will create an RPM on Ubuntu. If that
+        # flag is not set to any value in cmake, then we create
+        # only .deb artifact on Ubuntu.
+        # Note that package alien must already be installed for
```

Comment:
The comment makes it sound as if setting -DCREATE_RPM_FOR_UBUNTU would build both .rpm and .deb while not setting it "only" builds .deb.

But looking at the code itself, only one or the other is built but never both. Assuming this is intentional, I would suggest removing "only" from the comment for clarity.